main.py

```diff
@@ -17,7 +17,7 @@
     room = '10'
 
 
-exit_words = [':q','exit','quit']
+exit_words = [':q','exit','quit','q']
 referer_header={'Referer': url}
 balance = ''
```

Comment:
```suggestion
balance = float()
```

Maybe initialise the balance to float instead of str, since we're expecting it to be float after the casts you introduce.

Reply:
As long as the balance is similarly converted every time it is fetched from the backend, that should work!